packages/parse/src/index.test.ts

```diff
@@ -148,6 +148,17 @@ describe("parsing a changeset", () => {
       summary: ""
     });
   });
+  it("should be fine if the changeset only contains frontmatter", () => {
+    const changesetMd = outdent`---
+    ---
+    `;
+
+    const changeset = parse(changesetMd);
```

Comment:
`outdent` removes a trailing newline by default and all but it feels a little bit implicit here. I would prefer if we could do this explicitly here:
```suggestion
    const changeset = parse('---\n---');
```

Reply:
I totally overlooked that you have `outdent`. Updated!